src/PrestaShopBundle/Form/Admin/Type/IconButtonType.php

```diff
@@ -48,6 +48,8 @@ public function configureOptions(OptionsResolver $resolver)
             ->setDefaults([
                 'icon' => null,
                 'type' => 'button',
+                // @todo: this should by default be inherited from parent because of ModifyAllShopsExtension, but it doesn't work for some reason
+                'modify_all_shops' => false,
```

Comment:
For some reason it fails to inherit parent options (I tried adding ButtonType into extended types of ModifyAllShopsExtension, but it didn't help)

Reply:
outdataed.  The option modify_all_shops shouldn't have been added to footer url preview (seems I accidentally added it). Problem is gone as none of those buttons need the extension anymore.